src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+* {
+  box-sizing: border-box;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 0.3s;
```

Comment:
Same with colors and the transition duration

Reply:
Done